kratos/geometries/quadrature_point_curve_on_surface_geometry.h

```diff
@@ -154,6 +154,14 @@ class QuadraturePointCurveOnSurfaceGeometry
             rOutput[0] = mLocalTangentsU;
```

Comment:
Changes in Kratos core geometries definitively need to be done in a separate PR to ease an eventual backtracking of the changes.

Reply:
Agree

Reply:
Without this additional feature in the quadrature_point_curve_on_surface_geometry.h, both the two conditions cannot compute the normal.

Do we have to first do a PR of this method?

Reply:
Yes. You first merge that method. Then you retake this PR by merging the (updated) master to this branch.

Reply:
Isn't that function returning the same as the Normal function?

Reply:
Thank you for pointing that out, Tobias. I wasn't using it because in 3D (_quadrature_point_surface_in_volume_geometry.h_), it wasn't working properly. However, this issue is specific to the 3D context, so it doesn't make sense to modify the curve_on_surface.

Now, I use the Normal method directly, without making any changes to the geometries.